docs/windows.md

```diff
@@ -0,0 +1,197 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay network to route packets between nodes.
```

Comment:
Guess change "route" to "forward".

But would you remove implementation/design (to line 42) from this doc if you will have another architecture doc?